src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "The number is: " + number + " the color is: " + color;
```

Comment:
The toString method is not formatted properly. It should return a string formatted as 'ball #{number} is {color}', not 'The number is: {number} the color is: {color}'.

Reply:
done